components/global/header/LinkToParentLibrary.tsx

```diff
@@ -0,0 +1,27 @@
+"use client"
+
+import React, { useContext } from "react"
+
+import SmartLink from "@/components/shared/smartLink/SmartLink"
+import { getEnv } from "@/lib/config/env"
+import { DplCmsConfigContext } from "@/lib/providers/DplCmsConfigContextProvider"
+
+const LinkToParentLibrary = () => {
+  const dplCmsConfig = useContext(DplCmsConfigContext)
+  const parentLibraryUrl = getEnv("DPL_CMS_HOSTNAME")
+
+  return (
+    <p className="text-typo-caption">
+      En del af{" "}
+      <SmartLink
+        className="animate-text-underline"
+        linkType="external"
+        href={parentLibraryUrl}
+        target="_blank">
+        {dplCmsConfig?.libraryInfo.name}
+      </SmartLink>
+    </p>
+  )
```

Comment:
What happens if we do not get the library name? As far as I can see that case is not being handled.

Reply:
I was expecting it to always be present, but good point. I changed it a bit now :)